File: AndroidPrac2/app/src/main/java/com/antilog/androidprac2/MainActivity.java
package com.antilog.androidprac2;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;

public class MainActivity extends AppCompatActivity {

    EditText et_id;
    Button btn_button;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        //onCreate 는 어플을 처음 실행하였을때 처음으로 실행하는 생명주기
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        //해당 xml 파일을 연결해서 실행

        et_id = findViewById(R.id.et_id);
        btn_button = findViewById(R.id.btn_button);

        btn_button.setOnClickListener(new View.OnClickListener(){

            @Override
            public void onClick(View v) {
                et_id.setText("im jinsu");
                //setText 는 엑션으로 값을 바꾸어줄때 사용한다.

            }
        });
    }
}

File: SOPT_Semina/FristSemina/app/src/main/java/com/example/firstsemina/MainActivity.kt
package com.example.firstsemina

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.EditText
import android.widget.TextView
import android.widget.Toast

class MainActivity : AppCompatActivity() {

    val et_id = findViewById<EditText>(R.id.editText2)
    val et_pw = findViewById<EditText>(R.id.editText3)
    val btn_login = findViewById<Button>(R.id.button)
    val tv_register = findViewById<TextView>(R.id.textView4)

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        btn_login.setOnClickListener {
            if (et_id.text.isNullOrBlank() || et_pw.text.isNullOrBlank()) {
                Toast.makeText(this, "아이디와 비밀번호를 확인!", Toast.LENGTH_SHORT).show()
            } else {
                val intent = Intent(this, HomeActivity::class.java)
                startActivity(intent)
            }
        }

        tv_register.setOnClickListener {
            val intent = Intent(this, RegisterAcitvity::class.java)
            startActivity(intent)
        }
    }
}
